### PR TITLE
zombies are told to behave as zombies [remade & ready]

### PR DESCRIPTION
**Pull request by ghost**
## Changelog
:cl: zamolxius
tweak: made it so zombies are told to behave like a zombie, and not like a reskinned human
add: gives zombies aphasia and removes galactic common language.
/:cl:

## Why It's Good For The Game
people usually just go ask people to cure them, not knowing it's shit, 25TC goes to waste. might as well remove romerol if people just get themselves cured

download (2)
it's actually deciseconds.

zombies can't even talk to regular people, but if you are  gutsy enough you can get addicted to krokodil, survive the addiction, get brain damaged enough and blend in to survive

### CODE REVIEW

**Comment by Majick1** [comment]
+1, if zombies actually were just asking people to cure them then it's needed for the time being. I'd fix up the decisecond thing though, people may just reghost or log off if they see they're on a 8 minute respawn timer.

**Comment by ghost** [comment]
i fixed it up, it says deciseconds

**Comment by mikolaj007** [comment]
-1 because i fucking hate being a zombie
+2 because its a really needed change which, while i hate, is needed for romerol to work

File: hippiestation/code/modules/zombie/organs.dm [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	if(owner.stat != DEAD && !converts_living)
+		return
+	if(!owner.getorgan(/obj/item/organ/brain))
+		return
+	var/revive_time = rand(revive_time_min, revive_time_max)
+	if(!iszombie(owner))
+		to_chat(owner, "<span class='cultlarge'>You can feel your heart stopping, but something isn't right... \
+		life has not abandoned your broken form. You can only feel a deep and immutable hunger that \
+		not even death can stop, you will rise again!</span>")
+		to_chat(owner ,"<span class='big bold'> Do not seek to be cured, do not help any non-zombies in any way, and do not harm your zombie brethren. You are a creature of hunger and violence. You will revive in [revive_time] deciseconds. </span>")
```
Comment (ghost) [inline hippiestation/code/modules/zombie/organs.dm]: ```You will revive in [DisplayTimeText(revive_time)] seconds```

Comment (ghost) [reply hippiestation/code/modules/zombie/organs.dm]: thank you bruh

**Comment by bigbluebeaker** [comment]
+1 because romerol is expensive, people should ahelp if they don't want an antag role so someone else can play it

**Comment by ghost** [comment]
At least 1 approving review is required to merge this pull request.

**Review by Jujumatic — APPROVED** [review]

**Comment by Jujumatic** [comment]
uh sure why not

**Comment by ghost** [comment]
aight i'm just gonna enable DNR when I get killed by zombies then

**Comment by ghost** [comment]
there's no DNR on non baycode-nonparacode servers @steamp0rt

**Comment by mikolaj007** [comment]
but there is, look at discord #coding-copter

**Comment by ghost** [comment]
merge it already

**Comment by ghost** [comment]
@zamolxius yes there is, stupid

**Pull request merged**